compiler/rustc_ast/src/visit.rs

```diff
@@ -19,20 +19,20 @@ use crate::token;
 use rustc_span::symbol::{Ident, Symbol};
 use rustc_span::Span;
 
-#[derive(Copy, Clone, PartialEq)]
+#[derive(Copy, Clone, Debug, PartialEq)]
```

Comment:
Are these `Debug`s necessary?

Reply:
No they are not, but I was under the impression that things that *can* be `#[derive(Debug)]` should be, for ease of use for other people.  I had to add the derive to see some of the stuff that was passed to the ast early lint when I had an early pass version.